scripts/ingest_docs.py

```
@@ -85,16 +137,29 @@ def main():
-    # Prepare data for ChromaDB
+    # Prepare data for RAG
     documents = chunks
     ids = [f"{file_path.stem}_{i}" for i in range(len(chunks))]
     metadatas = [{"source": file_path.name, "chunk_index": i} for i in range(len(chunks))]
 
-    # Add to RAG
-    if rag_service.add_documents(documents, metadatas, ids):
-        logger.info("✅ Successfully ingested %s", file_path.name)
-    else:
-        logger.error("❌ Failed to ingest document.")
+    async def ingest():
```

Ruff (0.14.10)

Missing return type annotation for private function `ingest`

Add return type annotation: `None`

(ANN202)